modules/devguide/examples/java/src/main/java/com/couchbase/devguide/CloudConnect.java

```diff
@@ -43,7 +43,8 @@ public static void main(String... args) {
 
         ClusterEnvironment env = ClusterEnvironment.builder()
                 .securityConfig(SecurityConfig.enableTls(true)
-                        .trustManagerFactory(InsecureTrustManagerFactory.INSTANCE))
+                        // .trustManagerFactory(InsecureTrustManagerFactory.INSTANCE))  // use in testing if you use .trustManagerFactory(InsecureTrustManagerFactory.INSTANCE)) instead of Cert Auth (below)
```

Comment:
same

Reply:
more accurate would be "used in testing if you want to trust all server certificates  / do not validate them"